packages/playwright-test/src/runner.ts

```diff
@@ -294,13 +294,7 @@ export class Runner {
         filterSuite(rootSuite, () => false, test => shardTests.has(test));
         total = rootSuite.allTests().length;
       }
-
-      if (process.stdout.isTTY) {
-        console.log();
-        const jobs = Math.min(config.workers, testGroups.length);
-        const shardDetails = shard ? `, shard ${shard.current} of ${shard.total}` : '';
-        console.log(`Running ${total} test${total > 1 ? 's' : ''} using ${jobs} worker${jobs > 1 ? 's' : ''}${shardDetails}`);
```

Comment:
This was deliberately outside of the scope of the reporter. Why do we want to differentiate terminal and non-terminal reporters? Do we also want to differentiate stdio vs terminal/tty?